### PR TITLE
Add DNS Search using xe pif-param-set

### PR DESCRIPTION
**Pull request by Forza-tng**
> Before submitting the pull request, you must agree with the following statements by checking both boxes with a 'x'.
> * [X] "I accept that my contribution is placed under the CC BY-SA 2.0 license [1]."
> * [] "My contribution complies with the Developer Certificate of Origin [2]." 
>
> [1] https://creativecommons.org/licenses/by-sa/2.0/
> [2] https://xcp-ng.org/docs/contributing.html#developer-certificate-of-origin-dco

### CODE REVIEW

**Review by benjamreis — CHANGES_REQUESTED** [review]
Hi!

Thanks for your contribution!
We require all contributors to [sign off](https://xcp-ng.org/docs/contributing.html#developer-certificate-of-origin-dco) their commits.

you can run 
```
git commit --amend --signoff
``` 
and then
```
git push -f
```
To add a sign off to your commits. :)

**Comment by Forza-tng** [comment]
> 



> Hi!
> 
> Thanks for your contribution! We require all contributors to [sign off](https://xcp-ng.org/docs/contributing.html#developer-certificate-of-origin-dco) their commits.
> 
> you can run
> 
> ```
> git commit --amend --signoff
> ```
> 
> and then
> 
> ```
> git push -f
> ```
> 
> To add a sign off to your commits. :)

I do not use git, but only github webui. I was not sure how to add that, so I didn't :) Can you help with this?

File: docs/networking.md [hunk 1]
```diff
@@ -317,6 +317,25 @@ This setup will save you costs of 2 network switches you would otherwise have to
...
+* First identify the PIF used as management interface.
+```
+# xe pif-list host-name-label=xcpng-srv01 management=true
+uuid ( RO)                  : 76608ca2-e099-9344-af36-5b63c0022913
+                device ( RO): bond0
+    currently-attached ( RO): true
+                  VLAN ( RO): -1
+          network-uuid ( RO): cc966455-d5f8-0257-04a7-d3d7c671636b
+```
+* Take note of the `uuid` field and pass that to  `xe pif-param-set`
```
Comment (stormi) [inline docs/networking.md:333]:
```suggestion
* Take note of the `uuid` field and pass that to `xe pif-param-set`
```

**Comment by stormi** [comment]
Using git in command line could be useful to you in the future. 

If you want to do it using the web UI only, the easiest would be to start over. When asked to create the commit, add the sign off information at the bottom of the commit message description (not to be confused with the description of the pull request, where you the the Xs in the boxes)

**Pull request closed without merging**